addons/crm/static/tests/forecast_view_tests.js

```diff
@@ -52,6 +52,7 @@ QUnit.module("Views", (hooks) => {
                     },
                     records: [],
                 },
+                partner: {},
```

Comment:
?

Reply:
Just above, in the `bar` field definition:
```js
bar: {
    string: "Bar",
    type: "many2one",
    relation: "partner",
    store: true,
    sortable: true,
},
```
In the test `"Forecast graph view"`, the view is grouped by `bar`, causing an issue since the mock server now tries to sort records based on an non existant relation (= crash). It's also why I changed the way the `fields` are assembled in the mock server constructor: to allow this kind of shorthand model declaration

Reply:
ok :-)